src/views/Dissolutions/UploadAffidavit.vue

```diff
@@ -23,6 +23,17 @@ export default class UploadAffidavit extends Mixins(CommonMixin) {
   @Getter getAffidavitStep!: UploadAffidavitIF
   @Getter getShowErrors!: boolean
 
+  @Action setIgnoreChanges!: ActionBindingIF
+
+  /** Called when component is created. */
+  private created (): void {
+    // ignore data changes until page has loaded
+    this.setIgnoreChanges(true)
+    Vue.nextTick(() => {
+      this.setIgnoreChanges(false)
+    })
+  }
```

Comment:
Because this code was missing, trying to quit or reload the page after having made NO changes would prompt for unsaved changes.

Reply:
![image](https://user-images.githubusercontent.com/10002889/140173186-4bae20f4-a51c-4abe-9bbe-5118fb7e1119.png)

Reply:
![image](https://user-images.githubusercontent.com/10002889/140173251-b9b67bfd-fa33-4c17-a6ea-5cf280120508.png)